table/plain/plain_table_reader.cc

```diff
@@ -11,14 +11,15 @@
 #include <vector>
 
 #include "db/dbformat.h"
-
+#include "memory/arena.h"
+#include "monitoring/histogram.h"
+#include "monitoring/perf_context_imp.h"
```

Comment:
Ditto. I suggest to only change necessary part, in order to reduce merge upstream effort.